docs/installation/index.md

```diff
@@ -5,3 +5,13 @@
 * Clone the TransmogrifAI repo: `git clone https://github.com/salesforce/TransmogrifAI.git`
 * Build the project: `cd TransmogrifAI && ./gradlew compileTestScala installDist`
 * Start hacking
+
+# (Optional) Configuration
```

Comment:
I think we don't need to document output committer conf prominently since this is well documented for Hadoop and by extension for Spark itself. So probably not the time to modify index.md at this time either.